src/collision/mod.rs

```diff
@@ -260,51 +260,59 @@ pub struct Contacts {
     pub during_current_frame: bool,
     /// True if the bodies were in contact during the previous frame.
     pub during_previous_frame: bool,
```

Comment:
These and `is_sensor` should use a `ContactPairFlags` bitflag type or similar, but I'll do that in another PR where I rework contact pair management.